こちらのブランチにある`upload_to_hf.py`を利用してテストを兼ねていくつかモデルのアップロードを試してみました

[こちら](https://huggingface.co/llm-jp/7b-v1.0.1-63500step.code10K_en20K_ja30K_ver2.2-instruct-full-jaster)のモデルに関しては問題なくアップロードできたのですが、
llm-jpクラスタの`/model/7B_HF/llm-jp-7b-63500step.code10K_en20K_ja30K_ver2.2-instruct-lora-all-jaster`にあるモデルに関しては途中でエラーが出てアップロードできませんでした

```
uploading README.md to branch main ...
Traceback (most recent call last):
  File "/home/sosuke/.local/lib/python3.10/site-packages/huggingface_hub/utils/_errors.py", line 286, in hf_raise_for_status
    response.raise_for_status()
  File "/usr/lib/python3/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://huggingface.co/api/models/llm-jp/7b-v1.0.1-63500step.code10K_en20K_ja30K_ver2.2-instruct-lora-all-jaster-test/commit/main

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/model/sosuke/modelwg/upload_to_hf.py", line 49, in <module>
    main()

... 中略

    hf_raise_for_status(commit_resp, endpoint_name="commit")
  File "/home/sosuke/.local/lib/python3.10/site-packages/huggingface_hub/utils/_errors.py", line 329, in hf_raise_for_status
    raise BadRequestError(message, response=response) from e
huggingface_hub.utils._errors.BadRequestError:  (Request ID: xxxxxx)

Bad request for commit endpoint:
"base_model" with value "/model/7B_HF/llm-jp-7b-63500step.code10K_en20K_ja30K_ver2.2" is not valid. Use a model id from https://hf.co/models.
```

原因を調べたところ、README.md内のモデルカード情報の`base_model`のところにローカルのモデルパスが含まれてしまっていることが問題となっているようです

```md
---
library_name: peft
base_model: /model/7B_HF/llm-jp-7b-63500step.code10K_en20K_ja30K_ver2.2
---

# Model Card for Model ID

... 略 ...
```

@Taishi-N324 @hiroshi-matsuda-rit 
対処としてはおそらくREADMEを書き換えることでアップロード可能となると思いますが、今後の作業効率化を踏まえてなにかより賢い方法など知っていれば教えていただきたいです

--- a/README.md
+++ b/README.md
@@ -33,7 +33,8 @@ Example:
 ### Upload to HF
 
 ```console
-python upload_to_hf.py /model/7B_HF/llm-jp-7b-63500step.code10K_en20K_ja30K_ver2.2/ llm-jp/7b-v1.0.1-63500step.code10K_en20K_ja30K_ver2.2 main
+$ source Megatron-Llama2/venv/bin/activate
+$ python upload_to_hf.py /model/7B_HF/llm-jp-7b-63500step.code10K_en20K_ja30K_ver2.2/ llm-jp/7b-v1.0.1-63500step.code10K_en20K_ja30K_ver2.2 main
 ```
 
 ## Supervised Fine-tuning with llm-jp-sft
@@ -60,8 +61,8 @@ $ source venv/bin/activate
 
 For Llama models:
 ```console
-$ CUDA_VISIBLE_DEVICES=0 mdx/train_peft_single_gpu.sh /model/7B_HF/model_name/ /model/7B_HF/model_name/ dataset/ mdx/dataset_jaster.sh 5 /model/7B_HF/model_name-jaster-lora-all 1 64 --peft_target_model llama-all
-$ CUDA_VISIBLE_DEVICES=0 mdx/train_peft_single_gpu.sh /model/7B_HF/model_name/ /model/7B_HF/model_name/ dataset/ mdx/dataset_gpt4_self_inst_ja.sh 5 /model/7B_HF/model_name-self-inst-lora-all 1 64 --peft_target_model llama-all
+$ CUDA_VISIBLE_DEVICES=0 mdx/train_peft_single_gpu.sh /model/7B_HF/model_name/ /model/7B_HF/model_name/ dataset/ mdx/dataset_jaster.sh 5 /model/7B_HF/model_name-jaster-lora-all 2 32 --peft_target_model llama-all
+$ CUDA_VISIBLE_DEVICES=0 mdx/train_peft_single_gpu.sh /model/7B_HF/model_name/ /model/7B_HF/model_name/ dataset/ mdx/dataset_gpt4_self_inst_ja.sh 5 /model/7B_HF/model_name-self-inst-lora-all 2 32 --peft_target_model llama-all
 ```
 
 For GPT-2 models:

